ibm/provider/provider.go

```diff
@@ -1049,98 +1050,99 @@ func Provider() *schema.Provider {
+			"ibm_container_ingress_secret_tls":                   kubernetes.ResourceIBMContainerIngressSecretTLS(),
+			"ibm_container_ingress_secret_opaque":                kubernetes.ResourceIBMContainerIngressSecretOpaque(),
+			"ibm_container_cluster":                              kubernetes.ResourceIBMContainerCluster(),
+			"ibm_container_cluster_feature":                      kubernetes.ResourceIBMContainerClusterFeature(),
+			"ibm_container_bind_service":                         kubernetes.ResourceIBMContainerBindService(),
+			"ibm_container_worker_pool":                          kubernetes.ResourceIBMContainerWorkerPool(),
+			"ibm_container_worker_pool_zone_attachment":          kubernetes.ResourceIBMContainerWorkerPoolZoneAttachment(),
+			"ibm_container_storage_attachment":                   kubernetes.ResourceIBMContainerVpcWorkerVolumeAttachment(),
+			"ibm_container_nlb_dns":                              kubernetes.ResourceIBMContainerNlbDns(),
+			"ibm_container_dedicated_host_pool":                  kubernetes.ResourceIBMContainerDedicatedHostPool(),
+			"ibm_container_dedicated_host":                       kubernetes.ResourceIBMContainerDedicatedHost(),
+			"ibm_cr_namespace":                                   registry.ResourceIBMCrNamespace(),
+			"ibm_cr_retention_policy":                            registry.ResourceIBMCrRetentionPolicy(),
+			"ibm_ob_logging":                                     kubernetes.ResourceIBMObLogging(),
+			"ibm_ob_monitoring":                                  kubernetes.ResourceIBMObMonitoring(),
+			"ibm_cos_bucket":                                     cos.ResourceIBMCOSBucket(),
+			"ibm_cos_bucket_replication_rule":                    cos.ResourceIBMCOSBucketReplicationConfiguration(),
+			"ibm_cos_bucket_object":                              cos.ResourceIBMCOSBucketObject(),
+			"ibm_cos_bucket_object_lock_configuration":           cos.ResourceIBMCOSBucketObjectlock(),
+			"ibm_cos_bucket_website_configuration":               cos.ResourceIBMCOSBucketWebsiteConfiguration(),
+			"ibm_cos_bucket_lifecycle_configuration":             cos.ResourceIBMCOSBucketLifecycleConfiguration(),
+			"ibm_dns_domain":                                     classicinfrastructure.ResourceIBMDNSDomain(),
+			"ibm_dns_domain_registration_nameservers":            classicinfrastructure.ResourceIBMDNSDomainRegistrationNameservers(),
+			"ibm_dns_secondary":                                  classicinfrastructure.ResourceIBMDNSSecondary(),
+			"ibm_dns_record":                                     classicinfrastructure.ResourceIBMDNSRecord(),
+			"ibm_event_streams_topic":                            eventstreams.ResourceIBMEventStreamsTopic(),
+			"ibm_event_streams_schema":                           eventstreams.ResourceIBMEventStreamsSchema(),
+			"ibm_event_streams_schema_global_compatibility_rule": eventstreams.ResourceIBMEventStreamsSchemaGlobalCompatibilityRule(),
```

Comment:
is it possible to have small naming convention
ibm_event_streams_schema_global_compatibility_rule?

Reply:
I have changed the name to "ibm_event_streams_schema_global_rule".  All tests still pass.